src/com/dakl/Tests.java

```diff
@@ -43,7 +45,8 @@ void threeByThree(Team center_team, Team[] surrounding, Space center, boolean is
         }
     }
 
-
+    // REVIEW : javadoc comments to explain the purpose if needed
+    
```

Comment:
Overall, I really enjoyed the thoroughness of the test suite. 

For test cases that are more than a few lines long (that exercise maybe non-trivial scenarios), consider adding a javadoc that explains the scenario that is being tested.